### DISCUSSION TITLE
Any way to automate using MFCMAPI?

### DISCUSSION THREAD

**Discussion by Ringsaker** [microsoft/mfcmapi#502] [Q&A] [answered]
Is there any way to automate or import values using the tool?  Specifically, I use MFCMAPI every 2-3 days as Exchange constantly renames my accounts and I detest having the entire email address as the account name.  I'm hoping there's a way via script or batch file to quickly change the PR_DISPLAY_NAME value.

**Comment by stephenegriffin** [comment, marked as answer]
Nope - sorry! MrMAPI can automate reading, but does not support automating modifying.
